### PR TITLE
feat(component): implement copy-paste for worksheet

### PR DESCRIPTION
**Pull request by bc-apostoliuk**
## What?

Implement copy/paste for worksheet

## Why?

To be able to copy/paste content through the columns.

## Screenshots/Screen Recordings

https://github.com/user-attachments/assets/c5d28852-7896-41a0-950d-02ce86416baf

## Testing/Proof

UT + Locally.

### CODE REVIEW

File: packages/big-design/src/components/Worksheet/hooks/useCopyPaste/useCopyPaste.ts [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    );
+
+    return cellIdxsToUpdate.map((idxToUpdate) => ({
+      rowIndex: idxToUpdate,
+      columnIndex: selectedCells[0].columnIndex,
+      disabled: selectedCells[0].disabled,
+      hash: selectedCells[0].hash,
+      type: selectedCells[0].type,
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+      value: rows[idxToUpdate][selectedCells[0].hash],
```
Comment (chanceaclark) [inline packages/big-design/src/components/Worksheet/hooks/useCopyPaste/useCopyPaste.ts:76]: 🍹 I'd see if we can remove this eslint-disable.

**Review by chanceaclark — APPROVED** [review]
Besides the 🍹 overall looks great!

**Review by jorgemoya — APPROVED** [review]
Nice!

File: packages/big-design/src/components/Worksheet/hooks/useCopyPaste/useCopyPaste.ts (hunk 1 above)
Comment (bc-apostoliuk) [reply packages/big-design/src/components/Worksheet/hooks/useCopyPaste/useCopyPaste.ts]: Done)

**Pull request merged**